plugins/sqs/src/plugin.cpp

```diff
@@ -51,6 +51,10 @@ class sqs_queue {
     if (auto endpoint_url = detail::getenv("AWS_ENDPOINT_URL_SQS")) {
       config.endpointOverride = *endpoint_url;
     }
```

Comment:
Can we actually remove this now? Not sure why this should be tied to an Arrow version instead of AWS_SDK_CPP.

Reply:
We tried removing it once when it already was supposed to be fixed and it didn't work.  

I'm happy to try again.

Reply:
Turns out this is surprisingly difficult to test, so we now just leave it in.